samples/subsys/power/stm32_pm/src/main.c

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2016 Intel Corporation.
+ * Copyright (c) 2019 STMicroelectronics.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "sample.h"
+
+struct device *gpio_bn, *gpio_led, *gpio_led2;
+
+/* in msec */
+#define WAIT_TIME 500
+#define SLEEP_TIME 4001
```

Comment:
Can you add a comment about those while ? why 4001 ?

Reply:
changed to set any timeout value